test/extended/apibinding/apibinding.go

```diff
@@ -79,4 +79,30 @@ var _ = g.Describe("[area/apiexports]", func() {
 		_, err = exutil.CreateResourceFromTemplate(k, cowboyTemplate)
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
+
+	g.It("Author:zxiao-Critical-[API] Verify that user can perform wildcard search via APIExport virtual workspace", func() {
+		g.By("# Create a test workspace")
+		k.SetupWorkSpace()
+
+		g.By("# Create custom resource cowboy using APIResourceSchema")
+		rsTemplate := exutil.FixturePath("testdata", "apibinding", "api_rs.yaml")
+		_, err := exutil.CreateResourceFromTemplate(k, rsTemplate)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Export custom resource using APIExport")
```

Comment:
@zimo-xiao should you say "create apibinding " instead ?

Reply:
Hi @kasturinarra, I have updated my test case. PTAL. Thanks!